packages/terra-doc-template/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-doc-template",
```

Comment:
This is late to the game, but is there a change we foresee including any other site/documentation templates in the future? This package contains two templates already. Wondering if we should name 'terra-templates' or 'terra-doc-template**s**' to reduce future confusion if we want to add more without adding a new package. Thoughts??

@bjankord @ryanthemanuel @mjhenkes @JakeLaCombe

Reply:
Personally, I'm fine with leaving it as singular. I know we have other components such as form and pagination that consists of multiple components, and I feel having it singular feels fine to me.

Reply:
+1 to leaving it singular